src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(BallColors.values().length);
```

Comment:
Creating a new instance of `Random` every time `getRandomColor` is called can be inefficient. Consider creating a single `Random` instance as a class field and reusing it.